src/ContentTypeRequest.jl

```diff
@@ -1,30 +1,26 @@
 module ContentTypeDetection
 
-import ..Layer, ..request
 using URIs
-using ..Pairs: getkv, setkv
 import ..sniff
 import ..Form
 using ..Messages
-import ..MessageRequest: bodylength, bodybytes
+import ..IOExtras
 import ..@debug, ..DEBUG_LEVEL
 
-abstract type ContentTypeDetectionLayer{Next <: Layer} <: Layer{Next} end
-export ContentTypeDetectionLayer
+export contenttypedetectionlayer
```

Comment:
I feel like these layer names are getting a bit lengthy to read without some way of breaking apart words such as,

> content_type_detection_layer